.github/workflows/ci.yml

```diff
@@ -41,6 +41,37 @@ jobs:
         node-version: [16]
         os: [ubuntu-latest, windows-latest]
         e2e-browser: ['chromium']
+    env:
+      TURBO_CACHE_KEY: ${{ matrix.os }}-${{ matrix.node-version }}
+      KIT_E2E_BROWSER: ${{matrix.e2e-browser}}
+    steps:
+      - run: git config --global core.autocrlf false
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2.2.4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: pnpm
+      - run: pnpm install --frozen-lockfile
+      - run: pnpm playwright install ${{ matrix.e2e-browser }}
+      - run: pnpm test
+      - name: Archive test results
+        if: failure()
+        shell: bash
+        run: find packages -type d -name test-results -not -empty | tar -czf test-results.tar.gz --files-from=-
+      - name: Upload test results
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          retention-days: 3
+          name: test-failure-${{ github.run_id }}-${{ matrix.os }}-${{ matrix.node-version }}-${{ matrix.e2e-browser }}
+          path: test-results.tar.gz
+  Cross-browser-test:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
my thinking was that we need to run the whole test suite in 16 since it's more likely that an incompatibility with 16 will slip in than an incompatibility with 18. unless you think we need to run the whole suite in both 16 and 18?

Reply:
I agree that I'm more worried about incompatibilities with 16 since it's the older version. I think for 18 I would either not run it at all or run everything to catch potential server side issues. I'd be fine either way. Running Chrome on Linux is pretty fast, so it won't be the last to finish if we do run Node 18 against everything.